miniz_oxide/src/inflate/core.rs

```diff
@@ -672,6 +672,138 @@ fn start_static_table(r: &mut DecompressorOxide) {
     memset(&mut r.tables[DIST_TABLE].code_size[0..32], 5);
 }
 
+/// Equivalent to (0..1024).map(|n| n.reverse_bits())
+static REVERSED_BITS_LOOKUP: [u32; 1024] = [
```

Comment:
I'm a bit skeptical about the usefulness vs. the overhead of including this table. This improves my benchmark using the image in #82 by about 13%.

I would be comfortable closing this PR if the maintainers feel it does not add a sufficient improvement. This is somewhat of a proof-of-concept. If this change does merge, this constant should be moved to a separate file.

Reply:
Yeah bumping to 1.46 should be fine I think